src/backend/InvenTree/plugin/builtin/integration/part_notifications.py

```diff
@@ -0,0 +1,80 @@
+"""Core set of Notifications as a Plugin."""
+
+from django.utils.translation import gettext_lazy as _
+
+import structlog
+
+import common.models
+import common.notifications
+import InvenTree.helpers
+import InvenTree.helpers_email
+import InvenTree.helpers_model
+import InvenTree.tasks
+from part.models import Part
+from plugin import InvenTreePlugin
+from plugin.mixins import EventMixin, SettingsMixin
+
+logger = structlog.get_logger('inventree')
+
+
+class PlgMixin:
+    """Mixin to access plugin easier.
+
+    This needs to be spit out to reference the class. Perks of python.
+    """
+
+    def get_plugin(self):
+        """Return plugin reference."""
```

Comment:
Is this code used anywhere?

Reply:
For some internal functions of my plugin updrate but not in core, will remove